modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -348,6 +353,13 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
 
     Hooks.postProcess(name, ed);
   }
+
+  if (!node) {
```

Comment:
Is there a reason why this can't be at the end of the `else` block around line 351?

Reply:
Yeah, I think it can be moved there.